js/plugins/firebase/package.json

```diff
@@ -40,9 +40,9 @@
     "@google-cloud/firestore": "^7.6.0",
     "firebase-admin": "^12.2.0",
     "firebase-functions": "^4.8.0 || ^5.0.0",
-    "@genkit-ai/ai": "workspace:*",
-    "@genkit-ai/core": "workspace:*",
-    "@genkit-ai/flow": "workspace:*"
+    "@genkit-ai/ai": "workspace:^0.5.7",
+    "@genkit-ai/core": "workspace:^0.5.7",
+    "@genkit-ai/flow": "workspace:^0.5.7"
```

Comment:
pnpm will translate these to the latest version from the workspace:
![image](https://github.com/user-attachments/assets/18b12643-8e20-42e3-b391-7035b6d28287)

it's not sufficient?

Reply:
That is most definitely sufficient.